### PR TITLE
feat: wire up rule machine

### PR DESCRIPTION
**Pull request by jaredsavage**

### CODE REVIEW

File: src/services/content.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import i18n from 'services/i18n'
+import contentManifest from 'content/manifest.json'
+
+export const getManifestFilesFromClasses = classes =>
```
Comment (vicrep) [inline src/services/content.ts]: Is classes a string or a list of strings?

File: src/components/results-bubble.tsx [hunk 2]
```diff
@@ -1,21 +1,32 @@
...
+    async function processSteps() {
+      const steps = Object.values(props.steps)
+      const answeredSteps = steps.filter(
+        (step: any) => step.value !== undefined
+      )
+      const classes = await getChatClassifications(answeredSteps)
+      setClasses(classes)
+    }
+    processSteps()
+  }, [props.steps])
```
Comment (vicrep) [inline src/components/results-bubble.tsx]: Is this just a derivation of props.state?

If so, consider using useMemo instead, it'll save you needing to maintain a state

File: src/services/content.ts (hunk 1 above)
Comment (jaredsavage) [reply src/services/content.ts]: list of strings

File: src/components/results-bubble.tsx (hunk 2 above)
Comment (jaredsavage) [reply src/components/results-bubble.tsx]: don't have time to look unfortunately

Comment (jaredsavage) [reply src/components/results-bubble.tsx]: never used it

File: src/services/content.ts (hunk 1 above)
Comment (vicrep) [reply src/services/content.ts]: A little `: string[]` would've been nice to clear that up 😇

**Pull request merged**